script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -217,7 +220,8 @@ The real question is, %{i}who was it from the Church that%{n}spread this informa
 [sha:976f31b97e08c740c1793be22d01ee96431cdb5b]{
 -- Page 533, Offset 17898.
 -- 生前のパーソナリティを維持したまま、吸血鬼見習いとして活動する不死者。
-These undead retain the personality they had when they were alive, and act as vampiric apprentices.
+// cadet to tie in with Arcueid's military ranking metaphors
+These undead retain the personality they had when they were alive, and act as vampire cadets.
```

Comment:
While I have no real problem reverting it (because actually, she uses the military metaphor later, so maybe this 吸血鬼見習い is more of her objective opinion), I will point out that 1. the feudal caste system was based on military titles and 2. the first 3 ranks are very much always described in terms of "soldiers" and what not.